activerecord/test/cases/relation/where_test.rb

```diff
@@ -441,7 +441,8 @@ def test_where_on_association_with_custom_primary_key_with_array_of_base
     end
 
     def test_where_on_association_with_custom_primary_key_with_array_of_ids
-      essay = Essay.where(writer: ["David"]).first
```

Comment:
This invalid test was added in https://github.com/rails/rails/commit/cd0ed12d1a9edc73be953d700748b5827df890c7.